### ISSUE TITLE
x/pkgsite: package removal request for gitlab.com/wbrc/progress

### ISSUE THREAD

**Issue by wbrc** [golang/go#60547]
### What is the path of the package that you would like to have removed?

gitlab.com/wbrc/progress

### Are you the owner of this package?

yes
 
### What is the reason that you could not retract this package instead?

I deleted the repo gitlab.com/wbrc/progress
Also I have another module with the same name on github and I want to avoid confusion

**Comment by bcmills** [comment]
Can you recreate the original repo path, publish a `v0.1.1` release retracting all versions up to and including itself, and then delete it again?

**Comment by wbrc** [comment]
I think I forgot to `go get` the new version. I re-published the module with a `v0.1.2` retracting any previous version. I also did a `go get`:
```bash
> go get gitlab.com/wbrc/progress@v0.1.2
go: downloading gitlab.com/wbrc/progress v0.1.2
go: upgraded gitlab.com/wbrc/progress v0.1.0 => v0.1.2
```

I think now it should work. Should I delete the repo or wait for now.

**Comment by bcmills** [comment]
I see the retraction posted on https://pkg.go.dev/gitlab.com/wbrc/progress, so I think you're good to delete the repo at this point. 👍
